go/ql/lib/semmle/go/frameworks/JWT.qll

```diff
@@ -0,0 +1,91 @@
+/** Provides models of commonly used functions in common JWT packages. */
+
+import go
+
+/**
+ * Provides models of commonly used functions in common JWT packages.
+ */
+module JWT {
+  /** Gets the package name `github.com/lestrrat-go/jwx` v2. */
+  string packageLestrrat() { result = package("github.com/lestrrat-go/jwx/v2/jwt", "") }
+
+  /** Gets the package name `github.com/lestrrat-go/jwx` v1. */
+  string packageLestrratv1() { result = package("github.com/lestrrat-go/jwx/jwt", "") }
```

Comment:
```suggestion
  string packageLestrratv1() { result = package("github.com/lestrrat-go/jwx", "jwt") }
```
"github.com/lestrrat-go/jwx" is the module path. "jwt" is the package path within the module.